server/models/sir_h/simulator.py

```diff
@@ -1,8 +1,15 @@
 from models.sir_h.state import State
 from models.rule import apply_rules, apply_evacuations
+from functools import lru_cache
+from frozendict import frozendict
 
 
-def run_sir_h(parameters, rules, data_chu=dict()):
+def run_sir_h(parameters, rules, data_chu=frozendict()):
```

Comment:
Is this function still used if you import `cached_run_sir_h` only, on server/app.py?

Reply:
this is to make the application compatible, in particular `run_sir_h` is still called from `labs/run_simulator` where parameters are not yet immuable